docs/src/examples.md

```diff
@@ -1,6 +1,184 @@
 # Examples
 
 ## 1D Column examples
+### Ekman Layer
```

Comment:
Please add your examples at the end of the Advection example, but before the 2D examples.

Also, please remove the root-level examples.md file that is not needed.